python-checks/src/test/resources/checks/afterJumpStatement.py

```diff
@@ -151,3 +151,8 @@ def with_stmt_raise(expect_raise, e):
     with expect_raise():
         raise e
     print("foo") # OK
+
+def return_inside_class():
+    class Foo:
+        return
+    print("foo") # OK
```

Comment:
If this is not valid python code, I am wondering what we are trying to deal with in this commit : what is the problem ?

Reply:
this is a unit test to assert we are not creating a CFG for this semantically incorrect code and, thus, avoiding having a FP at line 158

Reply:
I think this is worth mentioning in a comment above the testcase : something along the line of : "this is semantically incorrect python code, preventing the CFG generation, and so no issue is raised in this case"